src/react/src/components/AddLocationData.jsx

```diff
@@ -0,0 +1,336 @@
+            margin: '20px',
+            padding: '15px 25px 15px 25px',
+            display: 'center',
+            fontWeight: '900',
+            fontSize: '16px',
+            backgroundColor: theme.palette.action.main,
+            color: theme.palette.getContrastText(theme.palette.action.main),
+            '&:hover': {
+                backgroundColor: theme.palette.action.dark,
+            },
+        }),
+        container: Object.freeze({
+            backgroundColor: COLOURS.LIGHT_GREY,
+        }),
+        title: Object.freeze({
+            paddingLeft: '5%',
+            paddingRight: '5%',
+            paddingTop: '25px',
+            color: COLOURS.NEAR_BLACK,
+            fontWeight: 'bold',
+            marginBottom: '1rem',
+        }),
+        description: Object.freeze({
+            paddingLeft: '5%',
+            paddingRight: '5%',
+            paddingTop: '5px',
+            marginBottom: '2rem',
+            fontWeight: 'bold',
+        }),
+        dataOptions: Object.freeze({
+            display: 'flex',
+            paddingLeft: '5%',
+            paddingRight: '5%',
+            gap: '50px',
+            paddingBottom: '5%',
+            flexWrap: 'wrap',
+        }),
+        card: Object.freeze({
+            backgroundColor: COLOURS.WHITE,
+            boxShadow: 'none',
+            padding: '60px 25px 25px 25px',
+            width: '45%',
+            position: 'relative',
+            textAlign: 'center',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            overflow: 'hidden',
+        }),
+        cardTitle: Object.freeze({
+            fontSize: '32px',
+            margin: '0 auto',
+            maxWidth: '80%',
+            textAlign: 'center',
+            paddingTop: '15px',
+            paddingBottom: '15px',
+            fontWeight: '300',
+            lineHeight: '1.0',
+        }),
+        cardSub: Object.freeze({
+            fontSize: '16px',
+            margin: '0 auto',
+            maxWidth: '50%',
+            textAlign: 'center',
+            paddingBottom: '5px',
+            fontWeight: '600',
+        }),
+        cardRectangleView: Object.freeze({
+            position: 'absolute',
+            top: 0,
+            right: 0,
+        }),
+        cardSliceView: Object.freeze({
+            position: 'absolute',
+            bottom: -5,
+            right: 0,
+        }),
+        cardSliceDuoView: Object.freeze({
+            position: 'absolute',
+            top: 0,
+            right: '25%',
+        }),
+        cardIcon: Object.freeze({
+            color: COLOURS.NEAR_BLACK,
+            textAlign: 'center',
+            alignItems: 'center',
+        }),
+        highlight: Object.freeze({
+            color: COLOURS.NEAR_BLACK,
+            fontWeight: '600',
+        }),
+        messyData: Object.freeze({
+            backgroundColor: '#8428FA21',
```

Comment:
Please, move the color to the COLOURS.js which you have already imported.

Reply:
Done.